guardrails/guardrail_globals.py

```diff
@@ -135,3 +142,36 @@ def dead_code_exclude(self):
         if self.dead_code_ignore:
             cmd = "--exclude %s" % self.dead_code_ignore
         return cmd
+
+    def read_file_to_list(self):
+        """ Function to read ignored pylint files to list """
```

Comment:
Please take a call. If the doc-string exists because of a linting compulsion, we can remove the compulsion. The doc-string is not telling me anything more useful than the function name. May be we can rename the function to read_ignored_pylint_files_to_list() and be done with it - and remove the doc-string? 

Please consider this comment for other doc-strings too. No compulsion to change this with this version of pull-request though.

Reply:
Yes, the function name could be self explanatory. Just intend to keep it in consistent with the code and general coding practice hence planning to retain the comment(doc string)